src/data/faq.json

```
@@ -45,6 +45,16 @@
                             "<b>National:</b> 0800 7540002",
                             "<b>International:</b> +49 30 498 75402"
                         ]
+                    },
+                    {
+                        "title": "How do I create an error log?",
+                        "anchor": "error_log",
+                        "active": true,
+                        "textblock": [
+                            "With version 2.2, the Corona-Warn-App offers a new function for detailed technical problem analysis. The error log is intended to help the developers to faster perform a root cause analysis and to eliminate errors more efficiently.<br><b> Important:</b> Please report an issue <b>first</b> to the hotline (national: 0800 7540001 or international: +49 30 498 75401) to verify if there is already a known solution. An error log is only required if the CWA team asks for this report.",

```

Maybe link to https://www.coronawarn.app/de/faq/#international_phone_numbers instead of writing the phone numbers there to be consistent with https://www.coronawarn.app/de/faq/#who_can_help?

Also, use in-app wording "Error Report"

```suggestion
                            "With version 2.2, the Corona-Warn-App offers a new function for detailed technical problem analysis. The error report is intended to help the developers to faster perform a root cause analysis and to eliminate errors more efficiently.<br><b> Important:</b> Please report an issue <b>first</b> to the technical hotline (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>click here to see the phone numbers</a>) to verify if there is already a known solution. An error report is only required if the CWA team asks for this report.",
```

---

Good!